scripts/release_management/README.md

```diff
@@ -0,0 +1,65 @@
+# Release management scripts
+
+## Overview
+The scripts in this folder are used for release management in CircleCI. Although the scripts are fully configurable using input parameters,
+the default settings were modified to accommodate CircleCI execution.
+
+# Build scripts
+These scripts help during the build process. They prepare the release files.
+
+## bump-semver.py
+Bumps the semantic version of the input `--version`. Versions are expected in vMAJOR.MINOR.PATCH format or vMAJOR.MINOR format.
+
+In vMAJOR.MINOR format, the result will be patch version 0 of that version, for example `v1.2 -> v1.2.0`.
```

Comment:
when bumping major version happens?

Reply:
We currently do not bump the major version? We do not really follow https://semver.org and misuse the minor version for API breaking changes. I assume this will change as soon as we have a v1.0.0? But if I understand correctly the scripts can create such a release. It's just that major won't be bumped.

Reply:
The request was to keep version management in the developer's hand. vX.Y.Z, Z is automatically calculated using `bump-semver.py`. The rest is manual, If you open a v1.0 branch, the automation will trigger a v1.0.0 release. If you push something to that branch, automation will trigger a v1.0.1 release.